mbed-hal/spi_api.h

```diff
@@ -32,10 +32,33 @@
 #define SPI_FILL_WORD         (0xFFFFFFFF)
```

Comment:
As before the combination of those events (ie. ORed together).
But then its technically not an enum anymore, because `SPI_EVENT_ERROR | SPI_EVENT_RX_OVERFLOW` is not in the enum list.

Hey, I know: lets use C bit fields!1!!